infra/modules/azure_api_management/locals.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   apim = {
-    name           = "${module.naming_convention.prefix}-apim-${module.naming_convention.suffix}"
-    autoscale_name = var.tier == "l" ? "${module.naming_convention.prefix}-apim-as-${module.naming_convention.suffix}" : null
+    name           = module.naming_convention.name.api_management["1"]
```

Comment:
what does the '1' index means in this context?

Reply:
it is the first instance, so the one that will have `01` as a suffix, in the ReadMe I have inserted the explanation of what form the various values ​​will have, in short we have a name for each instance

Reply:
why do we hardcode `1` for this resource and dont' use `instance_number` ? do we expect to have just one instance for environment?

Reply:
yes the modules always create only one instance, but you are right about using `istance_number`, it is not necessarily always 1, I will fix it thanks